### DISCUSSION TITLE
How to cluster based on different Hashtags (HTO) ?

### DISCUSSION THREAD

**Discussion by ahmadsam66** [satijalab/seurat#4251] [Q&A] [answered]
Hello,

I have recently used HTO demultiplexing.  I have used this tutorial : https://satijalab.org/seurat/archive/v3.1/hashing_vignette.html

Now, after selecting singlet how can I separate them based on different Hashtags ? 

I used Label Hashtag1, Hashtag2 and Hashtag3. How can I get them using subset?

Thank you.

**Comment by timoast** [comment, marked as answer]
If you want to cluster your data based on the HTO counts you can change the default assay to the HTO assay, process the HTO counts as shown in the hashing vignette, run PCA, create a neighbor graph (`FindNeighbors()`), and find clusters using the HTO neighbor graph (`FindClusters()`).

**Reply by ahmadsam66** [reply]
thank you so much, 

But, I want to divide my samples based on each hashtags (1/2/3), to make gene expression clustering, Is HTO count is similar to gene expression?

Is it possible to run PCA, FindNeighbors and FindClusters for each hash tags separately ?

thanks.

**Reply by timoast** [reply]
If you want to process the gene expression data for each hash separately, you can split the Seurat object by hash (use the `SplitObject()`function), and then change the assay for each object to "RNA," and follow the workflow shown on the Seurat website for analyzing gene expression data.
